src/iuse.cpp

```diff
@@ -2104,6 +2112,18 @@ int iuse::sew_advanced(player *p, item *it, bool, const tripoint& )
     // Get the id of the material used
     const auto &repair_item = mod_materials[choice];
 
+    /*
+    DAMAGE+DESTROY
+    WASTED
+    MODIFY_WASTED
+    MODIFY
+    */
+
+    bool consume_mats = true;
+    bool consume_reqs = true;
```

Comment:
`consume_reqs` looks `true` everywhere.
`consume_mats` has one case where it is `false`. Still, in all the other cases you overwrite the `true` with another `true`, which is 3 extra lines that do nothing.